ui/job-view/pushes/PushJobs.jsx

```diff
@@ -108,7 +108,7 @@ class PushJobs extends React.Component {
     } = this.props;
 
     return (
-      <div role="button" tabIndex={0} onMouseDown={this.onMouseDown}>
+      <div role="list">
```

Comment:
Actually, the child of this div is a table, which is already enough. Adding the "list" role here would be invalid, since the table child is not a list item. Just remove the role "button", making this an ordinary container, and apply the suggestion by @camd above. The individual buttons inside the table that reference each item are already interactive, so the div itself is, for a11y purposes, not needed with any widget role.